code/dce-downloader/node/README.MD

```diff
@@ -0,0 +1,233 @@
+# DCE Downloader Tools in Node.js
+
+Two Node.js scripts files are provided to get data from the Bazaarvoice Displayable Content Export service. Please refer to [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
+
+## Prerequisite
+1. You should have Node.js 5.0 or later installed.
+2. You should install the package dependencies
+3. You must have a config.json file (see [Common Prerequisites](../README.MD#Common-Prerequisites))
+    
+## Development
+
+To get some background on working with Python, check out [this blog post](http://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/) and [this virtualenv doc](http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper).
+
+### MacOS Environment
+
+In your terminal, execute the following:
+
+```bash
+# Install node
+brew update
+brew install node
+```
+
+Install DCE package dependencies:
+
+```bash
+cd code/dce-downloader/node
+npm install --save
+```
+
+## dce.js
+
+This script is used to get dates/versions available and get individual files. This demonstrates the basic usage of the Displayable Content Export, including how to create the HMAC signature, the HTTP headers to include, and the manifest list and manifest file contents.
+
+Usage:
+
+  ```
+  dce.js --env <env> --path <path> --dest <dest> --config <config>
+
+  <config>   : config file (optional, defaults to ../config.json) 
+  <env>      : environment in config file to use (required)
+  <path>     : path of file to download (optional, manifests is downloaded in the absence of path)
+  <dest>     : local directory in which to save the downloaded data files (optional)
+  ```
+
+1. Get available command-line options
+
+   ```
+   dce.js
+   ```
+   
+1. Get manifests (dates/versions) available
```

Comment:
Ah I see. I'll "Start a Review" if I have future comments, I forgot that this wasn't Gerrit, and immediately publishes comments